app/views/extractorMetrics.scala.html

```diff
@@ -0,0 +1,48 @@
+@(extractorName: String, average: String, lastweeksubmitted: Int, lastmonthsubmitted: Int)(implicit user: Option[models.User])
+@import api.Permission
+
+@main("Extractor Details") {
+    <script src="@routes.Assets.at("javascripts/deleteUtils.js")" type="text/javascript"></script>
+    <script src="@routes.Assets.at("javascripts/extractors/extractors.js")" type="text/javascript"></script>
+    <style>
+            strong        { text-align: right; }
+            fieldset      { margin-top: 5px; }
+            .form-actions { margin-top: 15px; }
+    </style>
+
+    <ol class="breadcrumb">
+        <li><span class="glyphicon glyphicon-fullscreen"></span> <a href="@routes.Extractors.selectExtractors()">Extractor Catalog</a></li>
+        <li><span class="glyphicon glyphicon-search"></span> @extractorName</li>
+    </ol>
+
+    <div class="page-header"><h1>@extractorName Metrics Details</h1></div>
+
+    <div class="row">
+        <div class="col-sm-8">
+            <form>
```

Comment:
A form is usually used when we have controls to modify some state. In this case since you are just showing information it might be easier (with less HTML) use a simple row and col divs.

Reply:
fixed

Reply:
fixed